selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/EditorValidationTest.java

```diff
@@ -71,10 +71,10 @@ public void autoCompleteClassInTheSamePackage() {
         PROJECT_NAME + "/src/main/java/org/eclipse/qa/examples/AppController.java");
     editor.waitActiveEditor();
     // validation warnings
-    editor.waitAllMarkersDisappear(ERROR_MARKER);
```

Comment:
Why it was removed? It was an important check - ensure that there are no error markers just after the **AppController.java** has opened, wasn't it?

Reply:
As you see this check is not removed, it is moved to another place, this is related to test logic

Reply:
Another place - is after the start of editing Che, and check of absence of warning just after the file opened IMHO is also important thing and also ought to be checked somewhere. Do we have similar check in another test?

Reply:
By the way - I reviewed last 10 test executions here https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/ and can't find failures because of method **waitAllMarkersDisappear()** in which you removed to make the test stable.
Could you, please, write a link to the report of the error in the method **waitAllMarkersDisappear** during the test execution on CI?